specification/workloads/resource-manager/Microsoft.Workloads/connectors/preview/2023-10-01-preview/connectors.json

```diff
@@ -0,0 +1,1098 @@
+        }
+      ],
+      "properties": {
+        "identity": {
+          "$ref": "../../../common-types/v1/commonTypes.json#/definitions/UserAssignedServiceIdentity"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ConnectorProperties"
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "ConnectorList": {
+      "description": "Defines the collection of connector resources.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "description": "Gets the list of connector resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Connector"
+          }
+        },
+        "nextLink": {
+          "description": "Gets the value of next link.",
+          "type": "string",
+          "format": "uri"
+        }
+      }
+    },
+    "ACSSBackupConnectionList": {
+      "description": "Defines the collection of backup connection resources of virtual instances for SAP.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "description": "Gets the list of backup connection resources of virtual instances for SAP.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ACSSBackupConnection"
+          }
+        },
+        "nextLink": {
+          "description": "Gets the value of next link.",
+          "type": "string",
+          "format": "uri"
+        }
+      }
+    },
+    "NewRecoveryServicesVault": {
+      "x-ms-discriminator-value": "New",
+      "description": "New recovery services vault.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryServicesVaultProperties"
+        }
+      ],
+      "required": [
+        "name",
+        "resourceGroup"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the recovery services vault has to be created."
+        },
+        "resourceGroup": {
+          "type": "string",
+          "description": "The name of the resource group where the recovery services vault has to be created."
+        }
+      }
+    },
+    "ExistingRecoveryServicesVault": {
+      "x-ms-discriminator-value": "Existing",
+      "description": "Existing recovery services vault.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryServicesVaultProperties"
+        }
+      ],
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The resource ID of the recovery services vault that has been created."
+        }
+      }
+    },
+    "RecoveryServicesVaultProperties": {
```

Comment:
>RecoveryServicesVaultProperties [](http://example.com/codeflow?start=5&length=31)

Its child types are *ServicesVault and its usage/reference elsewhere makes it sound like that there shouldn't be a Properties suffix. Please review one of the generated SDKs to see whether the generated class names look reasonable.
Marking the comment as non-blocking though.